File: 151-reverse-words-in-a-string/reverse-words-in-a-string.ts
function reverseWords(s: string): string {
    let curr = s.split(" ").filter(word => word!="")
    console.log(curr);
    let l = 0;
    let r = curr.length -1;
    while (l<=r){
        let temp = curr[l]
        curr[l] = curr[r]
        curr[r] = temp
        l++
        r--
    }
    return curr.join(" ")
};

File: 151-reverse-words-in-a-string/reverse-words-in-a-string.py
class Solution:
    def reverseWords(self, s: str) -> str:
        new_s = s.split()
        l,r = 0, len(new_s)-1
        while (l<r):
            new_s[l],new_s[r] = new_s[r],new_s[l]
            l+=1
            r-=1
        return (" ").join(new_s)